database/models/user.go

```diff
@@ -17,6 +17,8 @@ type User struct {
 	Email    string `gorm:"unique_index" json:"email"`
 	Password string `json:"-"`
 
+	EmailVerified bool `json:"email_verified"`
```

Comment:
Since we add a field in the model, I think we should also add a field in the resource package, at least the `UserForAdmin` struct(and its converting function of course).
https://github.com/EduOJ/backend/blob/34e752396fbbf094516f98adeac6f06730bba076/app/response/resource/user.go#L32